UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.SubscriptionService/Helpers/CommonHelper.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.Extensions.Logging;
+using Polly;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using UKHO.ExternalNotificationService.Common.Logging;
+
+namespace UKHO.ExternalNotificationService.SubscriptionService.Helpers
+{
+    public static class CommonHelper
+    {
+        public static IAsyncPolicy<HttpResponseMessage> GetRetryPolicy(ILogger logger, int retryCount, double sleepDuration)
+        {
+            return Policy
+                .HandleResult<HttpResponseMessage>(r => r.StatusCode == HttpStatusCode.ServiceUnavailable)                
+                .OrResult(r => r.StatusCode == HttpStatusCode.InternalServerError)
+                .WaitAndRetryAsync(retryCount, (retryAttempt) =>
+                {
+                    return TimeSpan.FromSeconds(Math.Pow(sleepDuration, (retryAttempt - 1)));
+                }, async (response, timespan, retryAttempt, context) =>
```

Comment:
`context` is not used and can be discarded:
```suggestion
                }, async (response, timespan, retryAttempt, _) =>
```

Reply:
Done